apps/src/lib/tools/jsdebugger/DebugButtons.jsx

```diff
@@ -35,6 +37,44 @@ export default connect(
       canRunNext: PropTypes.bool.isRequired
     };
 
+    // Wrap button actions to add tracking of presses to investigate student use
+    // userInteracted tracks if the user has open/adjusted the debug console
+    togglePause() {
+      trackEvent(
+        'debug_commands',
+        'debug_button_press',
```

Comment:
do we want to have different a event for each command so we can also see which debug commands are used?

Reply:
I actually played around with this a little bit and I ended up deciding that since our question is "Are people using this?", just knowing any of the buttons were selected is equally useful. I think if we wanted to investigate, "How are people using this?", having individual events for the buttons would be a must.

Reply:
However, if it seems useful, I'm happy to make a quick change and track them individually.

Reply:
yeah, that makes sense. if we look at the data and decide we want/need more granularity, we can always update these events

Reply:
Suggested (the name needs some love, but you get the idea) :
```
function logAndContinue(callback) {
  trackEvent(
    'debug_commands',
    'debug_button_press',
    this.props.userInteracted
  );
  callback();
}

Reply:
^ Would allow you to reuse this function instead of having 4 separate function calls

Reply:
Oh shoot, I just saw this now after merging. Will add in follow-up PR.